velox/exec/fuzzer/WindowFuzzer.cpp

```diff
@@ -152,9 +154,7 @@ std::tuple<std::string, bool> WindowFuzzer::generateFrameClause() {
   auto frameStart = frameBound(startBoundOptions[startBoundIndex]);
   auto frameEnd = frameBound(endBoundOptions[endBoundIndex]);
 
-  return std::make_tuple(
-      frameTypeString + " BETWEEN " + frameStart + " AND " + frameEnd,
-      isRowsFrame);
+  return frameTypeString + " BETWEEN " + frameStart + " AND " + frameEnd;
```

Comment:
This should be a column containing the bounds for k-range frames, please refer to `WindowTestBase::rangeFrameTestImpl`.